.eslintrc.js

```diff
@@ -18,8 +18,16 @@ module.exports = {
     web3: true
   },
   rules: {
-    'no-console': ['off'],
-    'prefer-const': ['error'],
-    'semi': ['error', 'never']
+    "no-console": ["off"],
+    "prefer-const": ["error"],
+    semi: ["error", "never"],
+    quotes: [
+      "error",
+      "double",
```

Comment:
double quotes 😫

Reply:
@micahalcorn Do you prefer single quotes? I'm not opinionated on it, I just chose double because more of our code was using double. 😃

Reply:
I vote for single quotes too :-)